experimental/components/Calendar/CalendarCustomActionComponentRegistration.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.Dialogs.Debugging;
+using Microsoft.Bot.Builder.Dialogs.Declarative;
+using Microsoft.Bot.Builder.Dialogs.Declarative.Resources;
+using Microsoft.Bot.Solutions.Extensions.Actions;
+using Microsoft.BotFramework.Composer.CustomAction.Actions;
+using Microsoft.BotFramework.Composer.CustomAction.Actions.MSGraph;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+
+namespace Microsoft.BotFramework.Composer.CustomAction
```

Comment:
You might want to refactor the namespace now to match package id, or at least consider for another PR.

Reply:
Agreed. But this will be another PR. Changing namespace will most likely cause this PR to blow up thereby delaying things further. Since we are not shipping any of these I think it's okay for now. I'll add a work item to clean things up.